sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md

```diff
@@ -235,6 +236,12 @@ For the sample description and interaction instructions, please go [here](https:
 
 For the sample description and interaction instructions, please go [here](https://github.com/momuno/azure-sdk-for-c/blob/master/sdk/samples/iot/README.md#iot-hub-telemetry-sample).
 
+### IoT Hub Twin CBOR Sample
+
+- *Executable:* `paho_iot_hub_twin_cbor_sample`
+
+For the sample description and interaction instructions, please go [here](https://github.com/momuno/azure-sdk-for-c/blob/master/sdk/samples/iot/README.md#iot-hub-twin-cbor-sample).
```

Comment:
Would this link work? The master branch doesn't have `#iot-hub-twin-cbor-sample` so it just points to the top of the README.

Reply:
you're right.  It will work once the feature branch is merged into master.  What is the best approach on this?  If I change it to the branch its on, I might forget to update it later.

Reply:
Ultimately, it's fine to leave it as is.

>  If I change it to the branch its on, I might forget to update it later.

The master branch has a link verification step which _should_ guard against that, so it's probably not something you'd forget to update once your branch is merged into master. CI and nightly build steps will fail and  tell you.

Here is something you could explore, just to see what's feasible:
If the feature branch and CI has link verification turned on, you might not even be able to use a link pointing to a non-master branch. You could try pointing it to your feature branch and see if CI passes.

Reply:
good info, thank you i will check it out

Reply:
Here's a doc on links:
https://github.com/Azure/azure-sdk/blob/012ba1a8a12948e7e359837f1612b295f36f32a6/docs/policies/README-TEMPLATE.md#link-guidelines

@sima-zhu can help answer any questions about the link verification step. Sima, what's our general recommendation for checking in links when using feature branches? Do we still want them to point to master?

For example, here's the link verification step under `GenerateReleaseArtifacts`:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=848519&view=logs&j=a129effc-2dd1-54d1-fb5a-ad7bdc0e851d&t=7b4713e3-e110-5049-e231-cc8a1e664c68

The entry point is here:
https://github.com/Azure/azure-sdk-for-c/blob/e850c99ad9e1692c1b62d4a966c0fce945d123e8/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L385-L395

Reply:
The link verification script will fail if it is a new link in feature branch. That is expected. It is better to comment the link out for the first PR, and then have a new PR to comment it back. 

We expect to see people can take good care of the feature branch link before it is ready to check in to master.